pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -74,6 +74,12 @@
  * - broker added or removed from cluster
  */
 public class HelixExternalViewBasedQueryQuotaManager implements ClusterChangeHandler, QueryQuotaManager {
+
+  // Minimum 'working' value for app quota. If actual value is less than this (e.g. 0.0), it is considered as disabled.
+  private static final double MIN_APP_QUOTA = Math.nextUp(0.0);
```

Comment:
I think the intention here is to treat `0` as disabled. It is not very readable to have this minimum double, can we change the comparison (e.g. `<` to `<=`) sign instead?

Reply:
I changed the value and hid logic behind isDisabled(), isEnabled() methods.